### PR TITLE
Assignment4 @NazariyPogo

### PR DESCRIPTION
**Pull request by PlantBrush**
URL: https://lemon-ground-09e5fcf1e.1.azurestaticapps.net/

### CODE REVIEW

**Comment by ZackWard-student** [comment]
Working on a review

File: wordle-web/pages/index.vue [hunk 1]
```diff
@@ -12,6 +12,11 @@
                 Play
               </v-btn>
             </v-card-actions>
+            <v-card-actions class="justify-center">
+              <v-btn nuxt to="/game" color="primary" x-large width="200px">
```
Comment (ZackWard-student) [inline wordle-web/pages/index.vue:16]: Still just links to the game page, not an implementation of the word of the day. Missing the word of the day page.

File: wordle-web/pages/instructions.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            <h1>
+              <font class="success rounded"
+                ><span class="accent--text">CO</span></font
+              >
+              <font class="warning rounded"
+                ><span class="accent--text">LO</span></font
+              >
+              <font class="error rounded"
+                ><span class="accent--text">RS</span></font
+              >
```
Comment (ZackWard-student) [inline wordle-web/pages/instructions.vue:62]: Love that you made this dynamic with the themes!

**Review by ZackWard-student — CHANGES_REQUESTED** [review]
The instructions page looks great! When on a small screen the W icon should probably be smaller. The Dynamic sizing is pretty good as well on the existing pages!

There is no added theme.
Missing button for the instruction page in the hamburger menu.
Missing the word of the day page.
Missing the last 10 daily words with statistics page.

Looks like the backend was not updated at all...

File: wordle-web/layouts/default.vue [hunk 3]
```diff
@@ -33,17 +33,29 @@
...
-          >
+          <v-btn text block nuxt to="/leaderboard">
+            Leader Board <v-icon>mdi-equalizer</v-icon>
+          </v-btn>
         </v-list-item>
 
         <v-list-item>
-          <v-btn text block nuxt to="/leaderboard">
-            Leader Board <v-icon>mdi-equalizer</v-icon>
+          <v-btn text block nuxt to="/instructions">
```
Comment (mmwoodfo) [inline wordle-web/layouts/default.vue:42]: You also need a link to the instructions page here.

File: wordle-web/pages/index.vue (hunk 1 above)
Comment (mmwoodfo) [reply wordle-web/pages/index.vue:16]: Looks like this is still unresolved. Also looks like you guys didn't implement any of the API portions for this project.

**Pull request closed without merging**